README.md

```diff
@@ -10,6 +10,42 @@ Each time a tag is pushed for `latest` or `v*`; a distrubtion build is
 prepared in CI that will be pushed at `dist-<tagname>` tag.
 See `dist.yml` build action for details.
 
+```
+git tag -a v0.11 -m "my changes"
+git push origin v0.11 # will trigger a release build
+```
+Updating the dist-latest:
+```
+git tag -a -f latest -m "my changes"
+git push origin latest -f # need to force to override the tag
+```
+
+
+## Updating AndroidX
```

Comment:
I thought we were keeping GH instructions in CONTRIBUTING.md, but I'm happy to have the action docs here since it's also applicable.

Reply:
yea i put this here because it is more natural to check then androidx when someone wants to update this.